internal/compiler/typeregister.rs

```diff
@@ -91,9 +91,9 @@ pub(crate) const RESERVED_ACCESSIBILITY_PROPERTIES: &[(&str, Type)] = &[
     ("accessible-has-focus", Type::Bool),
     ("accessible-label", Type::String),
     ("accessible-value", Type::String),
```

Comment:
I'm actually not sure if the value needs to be a String or an float

Reply:
w3c calls these properties `aria-valuenow` `aria-valuetext`  `aria-valuemax` `aria-valuemin`
`aria-valuetext` is the text representation of valuenow. So i guess we just need one